.github/workflows/snapshot-release.yml

```diff
@@ -76,34 +81,86 @@ jobs:
       - name: Bump Package Versions
         id: changesets
         run: |
-          pnpm exec changeset version --snapshot ${{ steps.getSnapshotName.outputs.result }} > changesets.output.txt 2>&1
-          echo ::set-output name=result::`cat changesets.output.txt`
+          pnpm exec changeset status --output status.output.json 2>&1
+          pnpm exec changeset version --snapshot ${{ steps.getSnapshotName.outputs.result }} > version.output.txt 2>&1
+
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+
+          echo "version<<$EOF" >> $GITHUB_OUTPUT
+          echo "$(cat version.output.json)" >> $GITHUB_OUTPUT
+          echo "$EOF" >> $GITHUB_OUTPUT
+          cat version.output.json
```

Comment:
version.output.json did not get created for some reason

https://github.com/lilnasy/astro/actions/runs/6590750901/job/17907965624#step:12:26

Reply:
```suggestion
          echo "$(cat version.output.txt)" >> $GITHUB_OUTPUT
          echo "$EOF" >> $GITHUB_OUTPUT
          cat version.output.txt
```

Reply:
Good catch!